### ISSUE TITLE
Riot Launcher Path not found

### ISSUE THREAD

**Issue by thelanda** [Sky-CSC/OSR#36]
![Captura](https://github.com/Sky-CSC/OSL/assets/61026675/5aa157d6-30ed-426d-8f40-460d70f87d51)

Complete all the steps of the "OSL-Web", start the league of legends client, and then start the "OSL-Client" and now I get this error, where is the problem coming from?

**Comment by Sky-CSC** [comment]
Hi
Where is it installed ?

**Comment by thelanda** [comment]
> Hi Where is it installed ?

![image](https://github.com/Sky-CSC/OSL/assets/61026675/123c7469-8131-445e-892c-ca424bd5266e)


It is installed in a separate folder, in that path I have the 2 OSL folders (OSL Client and OSL Web)

To be more specific, I have the game installed on Disk D and I also have the "OSL" on Disk D

Following the instructions, start the Web first and then the Client

**Comment by Sky-CSC** [comment]
I'll note the bug and try to fix it.
Possibly from the launcher update or vanguard

**Comment by Sky-CSC** [comment]
I found the problem.
In `OSL-Client-1.2.0-win-x64/Configuration/riot.json` change the league process name `"leagueClientProcess": "LeagueClient"` by `"leagueClientProcess": "LeagueClientUX"`

**Comment by thelanda** [comment]
Well, it's solved!

Now I get a different error when opening the OSL Client:

![image](https://github.com/Sky-CSC/OSL/assets/61026675/e229d248-1a37-4f61-b233-3f6792e8d2c1)


2024-06-20 17:33:41 | [ERROR] [ReplayApi.RequestGameClientReplayAPI ] : Request to https://127.0.0.1:2999/replay/game failed, A connection cannot be established since the destination computer expressly denied such a connection. [::ffff:127.0.0.1]:2999 (127.0.0.1:2999)

Ignore this error and try to create and start a game in "Recruitment Tournament", the champ select and overlay in game do not work.

Basically I get "Warning! Champ Select not in progress"

How do I solve all this?

**Comment by thelanda** [comment]
Well, it's solved!

Now I have an error in champion selection.

The images of the bans appear but not the picks, what would be the solution?

![image](https://github.com/Sky-CSC/OSL/assets/61026675/af12e89e-3bdb-47d9-8b82-b4dbb6dd2342)

**Comment by Sky-CSC** [comment]
For version 1.2.0 and earlier, the patch assets do not download correctly. Copy the folder (patch number) to `OSL-Web-1.2.0-win-x64/wwwroot/assets/`.
Patch 14.15 : 
[Zip link](https://drive.google.com/file/d/12XW2arC68Oqa_3o3u72Rsy5bWLUrAvAe/view?usp=sharing)
[Folder link](https://drive.google.com/drive/folders/1g-FN7HYipVsvpg2CRrG89pEK3iaOwoW3?usp=drive_link)
